Tests/UnitTests/MediaContextTests.swift

```diff
@@ -0,0 +1,197 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPCore
+@testable import AEPEdgeMedia
+import XCTest
+
+class MediaContextTests: XCTestCase {
+    let mediaInfo = MediaInfo(id: "mediaID", name: "mediaName", streamType: "aod", mediaType: MediaType.Audio, length: 30.0, prerollWaitingTime: 0)!
+    let mediaMetadata = ["media.show": "sampleshow", "key1": "value1", "key2": "мểŧẳđαţả"]
+    let adBreakInfo = AdBreakInfo(name: "adBreakName", position: 1, startTime: 1.1)!
+    let adInfo = AdInfo(id: "adID", name: "adName", position: 1, length: 15.0)!
+    let adMetadata = ["media.ad.advertiser": "sampleAdvertiser", "key1": "value1", "key2": "мểŧẳđαţả"]
```

Comment:
Can we use the public constants for the main flow, eg. AdMetadataKeys.ADVERTISER,  AudioMetadataKeys.ARTIST?
We can still have tests with custom keys, but maybe add a separate test just for that